src/backend/utils/storage.js

```diff
@@ -89,7 +91,9 @@ module.exports = {
         'guid',
         post.guid,
         'feed',
-        post.feed
+        post.feed,
+        'user',
+        post.user
```

Comment:
I don't think we need to store the user in every post? Posts are already linked to a feed, so by having the user stored in the feed we could assum that the _owner_ of the feed _owns_ all the posts.

Reply:
@humphd I think @manekenpix is right, but I may not be thinking clearly. Any opinions?